experimental/examples/bubblebench/editable-shared-tree/README.md

```diff
@@ -0,0 +1,35 @@
+# @fluid-example/bubblebench-sharedtree
+
+## Getting Started
+
+You can run this example using the following steps:
+
+1. Run `npm install` and `npm run build:fast -- --nolint` from the `FluidFramework` root directory.
+   a. For an even faster build, you can add the package name to the build command, like this:
+   `npm run build:fast -- --nolint @fluid-example/bubblebench`
+1. Run `npm run start` from this directory (experimental/examples/bubblebench/sharedtree) and open <http://localhost:8080> in a web browser to see the app running.
+ <!-- AUTO-GENERATED-CONTENT:END -->
```

Comment:
something is wrong with this readme. There is no start block for the autogenerated content, and the autogenerated content is out of date (the package name in the instructions is incorrect)